### PR TITLE
Add conclusion

### PR DESCRIPTION
**Pull request by jzzheng22**
https://www.youtube.com/watch?v=dqONk48l5vY

### CODE REVIEW

**Review by daryllimyt — DISMISSED** [review]
dope LGTS

File: 013_conclusion/013_conclusion.tex [hunk 1]
```diff
@@ -1,3 +1,5 @@
 \chapter{Conclusion}\label{conclusion}
 
-Conclusion section.
+This project has involved the design and implementation of six different agents with various strategies to self-organise and solve the common resource management problem, where the common resource in this case is the scarcity of food available to agents in the tower.
```
Comment (SaraFFdez) [inline 013_conclusion/013_conclusion.tex]:
```suggestion
This project has involved the design and implementation of six different agents with various strategies to self-organise and solve a common resource management problem, where the common resource, in this case, is the scarcity of food available to agents in the tower.
```

File: 013_conclusion/013_conclusion.tex [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+
+It has been shown that in order for self-organisation to be achieved within the tower, communication is necessary between all present agents through both messaging and the signing of treaties. Agents which were succesful in achieving self-organisation did so by gaining knowledge of both the tower and other agents, and applying this knowledge appropriately. Without this, the most logical behaviour of an agent is to act purely selfishly, giving itself the best chance of survival in a random system.
```
Comment (SaraFFdez) [inline 013_conclusion/013_conclusion.tex]:
```suggestion
It has been shown that in order for self-organisation to be achieved within the tower, communication is necessary between all present agents through both messaging and the signing of treaties. Agents which were successful in achieving self-organisation did so by gaining knowledge of both the tower and other agents, and applying this knowledge appropriately. Without this, the most logical behaviour of an agent is to act purely selfishly, giving itself the best chance of survival in a random system.
```

**Review by JaafarRammal — DISMISSED** [review]
You shall not pass

**Review by MattSScott — COMMENTED** [review]
I think we need a reference to governance, sorry to be that guy

File: 013_conclusion/013_conclusion.tex [hunk 3]
```diff
@@ -1,3 +1,5 @@
 \chapter{Conclusion}\label{conclusion}
 
-Conclusion section.
+This project has involved the design and implementation of six different agents with various strategies to self-organise and solve a common resource management problem, where the common resource, in this case, is the scarcity of food available to agents in the tower.
+
+It has been shown that in order for self-organisation to be achieved within the tower, communication is necessary between all present agents through both messaging and the signing of treaties. Agents which were successful in achieving self-organisation did so by gaining knowledge of both the tower and other agents, and applying this knowledge appropriately. Without this, the most logical behaviour of an agent is to act purely selfishly, giving itself the best chance of survival in a random system.
```
Comment (MattSScott) [inline 013_conclusion/013_conclusion.tex]:
```suggestion
It has been shown that in order for self-organisation to be achieved within the tower, communication is necessary between all present agents through both messaging and the signing of treaties to simulate localised governance. Agents which were successful in achieving self-organisation did so by gaining knowledge of both the tower and other agents, and applying this knowledge appropriately. Without this, the most logical behaviour of an agent is to act purely selfishly, giving itself the best chance of survival in a random system.
```

Comment (MattSScott) [reply 013_conclusion/013_conclusion.tex]: this would probs do it

**Review by daryllimyt — DISMISSED** [review]
sleep time

**Review by SaraFFdez — APPROVED** [review]
LGTM

**Review by daryllimyt — APPROVED** [review]

**Pull request merged**